features/test-site/_includes/responsive-image.html

```diff
@@ -1 +1,3 @@
+{% assign largest = resized | sort: 'width' | last %}
```

Comment:
Is this change intentional?

Reply:
@wildlyinaccurate This line triggers the bug described in #103. I should have left some comment explaining the purpose, sorry about that.